### PR TITLE
added .gitattributes

### PR DESCRIPTION
**Pull request by rvitaliy**
excluded useless files for production

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,13 @@
+* text=auto
+
+/Tests export-ignore
+.doctrine-project.json export-ignore
+.gitattributes export-ignore
+.gitignore export-ignore
+.scrutinizer.yml export-ignore
+.travis.yml export-ignore
+Changelog.md export-ignore
```
Comment (alcaeus) [inline .gitattributes]: I'd opt for leaving this along with the readme, what do you think?

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1,13 @@
+* text=auto
```
Comment (alcaeus) [inline .gitattributes]: I think this should be safe but haven't used it before. Opinions @doctrine/doctrinecore?

Comment (ostrolucky) [reply .gitattributes]: yeah seems ok for this repo. it's all source code

**Comment by alcaeus** [comment]
Thanks @rvitaliy!

**Pull request merged**